tests/integration/test_bundle_deployment.py

```diff
@@ -33,17 +37,24 @@ async def test_bundle_deployment_works(self, ops_test: OpsTest, lightkube_client
             timeout=1500,
         )
```

Comment:
i think L28-L34 should be with the if block, in 1.9 we can just wait for all charms to be active together, no need to wait on istio individually

Reply:
True